meinberlin/assets/scss/style_user_facing.scss

```diff
@@ -1,55 +1,26 @@
 // The plan is, as development continue to slowely seperate the dashboard styling from the BO styling and
-// eventually have seperate stylsheet and load totally seperate styles for the user side (just BO styles
-// and our overwrites for a4 components) and dashboard styles (bootstrap and our component styles) some
-// files maybe shared like the colour variables but most will be seperated
+// eventually have seperate stylsheet and load totally seperate styles for the public pages (just BO styles
+// and our additional styles and overwrites) and dashboard styles (bootstrap and our component styles)
 // FIXME how to load header and footer styling?
+@import "~bootstrap/scss/functions"; // to be deleted after menues changes
 
-@import "~bootstrap/scss/functions";
+@import "styles_user_facing/variables";
 
-@import "mixins";
-@import "variables";
-
-@import "~bootstrap/scss/variables";
-@import "~bootstrap/scss/maps";
-@import "~bootstrap/scss/mixins";
-
-@import "~bootstrap/scss/containers";
-@import "~bootstrap/scss/grid";
-@import "~bootstrap/scss/forms";
-@import "~bootstrap/scss/buttons";
-@import "~bootstrap/scss/transitions";
-@import "~bootstrap/scss/dropdown";
-@import "~bootstrap/scss/close";
-
-@import "~bootstrap/scss/modal";
-@import "~bootstrap/scss/spinners";
-
-@import "~bootstrap/scss/helpers/visually-hidden";
-
-@import "../berlin_css/berlin_marketing";
-
-@import "base";
-@import "layout";
-@import "form";
+@import "~bootstrap/scss/variables"; // to be deleted after menues changes
+@import "~bootstrap/scss/maps"; // to be deleted after menues changes
+@import "~bootstrap/scss/mixins"; // to be deleted after menues changes
+@import "~bootstrap/scss/dropdown"; // to be deleted after menues changes
```

Comment:
need bits of bootstrap for dropdowns but can be deleted after filtering and other menus are changed